package.json

```diff
@@ -50,6 +50,7 @@
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.0.0",
+    "ember-maybe-import-regenerator": "^0.1.6",
```

Comment:
is this only used for tests?

Reply:
Yes. Was getting it for free with `ember-concurrency` apparently. Shouldn't add to the library size as it's just a `devDependency` but I can roll the tests back to nested `andThen`'s if that's preferred.

Reply:
Nope, that sounds great to me.  Just wanted to double check.